### PR TITLE
Update to new Fedora Bootstrap

### PR DESCRIPTION
**Pull request by ryanlerch**
I started the new Fedora Bootstrap (https://pagure.io/fedora-bootstrap) and these patches change the datagrepper to use the new version of Bootstrap.

### CODE REVIEW

**Comment by pypingou** [comment]
Stupid question, can I haz a screenshot ? :)

File: datagrepper/templates/raw.html [hunk 1]
```diff
@@ -14,21 +14,24 @@
...
         {% if dict.get('msg_id') %}
             <a href="{{url_for('msg_id')}}?id={{dict['msg_id']}}&is_raw=true&size=extra-large">
                 <span class="details-history">JSON</span></a>
         {% endif %}
+        </td>
         {% if 'is_raw' in dict %}
             <a href="#" onclick="history.go(-1)">Go Back</a>
         {% endif %}
-    <span class="datetime pull-right">
+    <td class="datetime pull-right">
```
Comment (ralphbean) [inline datagrepper/templates/raw.html]: Does this still need a ``.pull-right`` if it is a ``<td>`` table cell now?

**Comment by ralphbean** [comment]
# Before

datagrepper-before

# After

datagrepper-after

**Comment by ralphbean** [comment]
# The Feed

datagrepper-feed

# A single message

datagrepper-item

**Comment by ralphbean** [comment]
After staring at it for a minute, here are some thoughts:

datagrepper-feed

And...

datagrepper-item

**Comment by ryanlerch** [comment]
Okies will fix the table spacing, and the location of the line when the message preview is open. I followed the directions on the README, and imported the dump, and i assume because there are so many, that is why my test site takes so long to load the feed page. Is there any way to reduce the number of items in the database, just to make testing the feed page easier?

Also, with the menubar, id rather not move it up, as on the other pages (not the feed), it fills the entire navbar row (we are using the narrow page on those pages and regular width on the feed), and it would be super cramped having it at the top, especially at smaller window sizes.

cheers,
ryanlerch

**Comment by ryanlerch** [comment]
I also noticed in your screenshots that the fonts are not being used in the feed page, so i will look into that -- i think it is because it uses a different master-style template to the other pages, and i forgot to add the webfont imports on those pages :)

**Comment by ralphbean** [comment]
> Is there any way to reduce the number of items in the database

If you can connect to your database directly, you can probably drop lots of old ones with something like:

```sql
DELETE FROM messages
    WHERE timestamp < (CURRENT_TIMESTAMP - '2 months'::interval);
```

... although the exact syntax depends on which database server you're using.

**Comment by ryanlerch** [comment]
OKies, these patches should fix all the above things.

The one thing i couldnt test however was the pending messages coming through, because i couldnt get that working with my test instance.

cheers.
ryanlerch

**Comment by ryanlerch** [comment]
Okies, for the time being, i added the fonts to here at datagrepper, but in the future, we probably do want them on the global directory, so they are cached for users across different fedora sites. 

I also made a new pagure repo for the fonts and the CSS for when we do that in the future.
https://pagure.io/fedora-bootstrap-fonts

**Comment by ralphbean** [comment]
Awesome.  Thanks @ryanlerch!

**Pull request merged**